docs/update_benchmark_tables.py

```diff
@@ -196,15 +215,18 @@ def create_domain_table(
 ) -> pd.DataFrame:
     rows = [benchmark_result_to_domain_row(result, langs) for result in results]
     df = pd.concat(rows)
+    # remove duplicate rows (notably the number of datasets row)
+    df = df.loc[~df.index.duplicated(keep="first")]
```

Comment:
Wouldn't it be possible to just not create duplicate columns in the first place, or would that be too finicky? This seems like a bit of a hack.

Reply:
I agree. It is a bit of a hack. I wanted to make sure that it aligns when concatenating the models (will see if I can avoid it though)

Reply:
fixed in new commit